hack/local-up-cluster.sh

```diff
@@ -213,7 +213,7 @@ function start_apiserver {
       --admission_control="${ADMISSION_CONTROL}" \
       --address="${API_HOST}" \
       --port="${API_PORT}" \
-      --runtime_config=api/v1beta3 \
+      --runtime_config=api/v1 \
```

Comment:
Yeah, upstream master uses v1beta3 in local-up-cluster. I rebased from upstream to your branch a couple of days ago to fix some issues :)